src/api/bulk/bulk.utils.ts

```diff
@@ -1,20 +1,59 @@
 import { z } from 'zod';
-import { ResponseSchema } from '../../utils/zod_helpers';
+import { BulkCaptureCreateSchema } from '../../schemas/capture-schema';
+import { BulkCritterCreateSchema } from '../../schemas/critter-schema';
+import { LocationBulkCreateSchema } from '../../schemas/location-schema';
+import { MortalityCreateSchema } from '../../schemas/mortality-schema';
+import { CollectionUnitCreateBodySchema } from '../collectionUnit/collectionUnit.utils';
+import {
+  FamilyChildCreateBodySchema,
+  FamilyCreateBodySchema,
+  FamilyParentCreateBodySchema
+} from '../family/family.utils';
+import { MarkingCreateBodySchema } from '../marking/marking.utils';
+import { QualitativeCreateSchema, QuantitativeCreateSchema } from '../measurement/measurement.utils';
 
+const OptionalRecordArraySchema = z.array(z.record(z.string(), z.unknown())).optional();
+
+/**
+ * Pre parse bulk create schema
+ *
+ */
```

Comment:
If you think its worthwhile: 

Could add an extra line here to explain that this is just a shallow structural schema (or however you can word that) of `BulkShapeSchema`, and is only used for pre-validation, etc.

Just to link them together a bit. I guess they are already in the router, but just in case.

Reply:
The current comment already implies that, but maybe just some extra detail would be good*